tests/Fixtures/Maker/expected/can-create-factory-for-not-persisted-class-interactively.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Factory;
+
+use Zenstruck\Foundry\ModelFactory;
+use Zenstruck\Foundry\Proxy;
+use Zenstruck\Foundry\Tests\Fixtures\Object\SomeObject;
+
+/**
+ * @extends ModelFactory<SomeObject>
+ *
+ * @method        SomeObject|Proxy     create(array|callable $attributes = [])
+ * @method static SomeObject|Proxy     createOne(array $attributes = [])
+ * @method static SomeObject[]|Proxy[] createMany(int $number, array|callable $attributes = [])
+ * @method static SomeObject[]|Proxy[] createSequence(array|callable $sequence)
+ */
+final class SomeObjectFactory extends ModelFactory
+{
+    public function __construct()
+    {
+        parent::__construct();
+
+        // TODO inject services if required (https://symfony.com/bundles/ZenstruckFoundryBundle/current/index.html#factories-as-services)
+    }
+
+    protected function getDefaults(): array
+    {
+        return [
+            // TODO add your default values here (https://symfony.com/bundles/ZenstruckFoundryBundle/current/index.html#model-factories)
+            'dateTimeImmutableMandatory' => \DateTimeImmutable::createFromMutable(self::faker()->dateTime()),
+            'dateTimeMandatory' => self::faker()->dateTime(),
+            'floatMandatory' => self::faker()->randomFloat(),
+            'intMandatory' => self::faker()->randomNumber(),
+            'someMandatoryPropertyWithUnionType' => null, // TODO add value manually
+            'someOtherObjectMandatory' => null, // TODO add Zenstruck\Foundry\Tests\Fixtures\Object\SomeOtherObject value manually
+            'stringMandatory' => self::faker()->text(),
```

Comment:
```suggestion
            'stringMandatory' => self::faker()->text(),
```

`text()` could be too long, what about using `->sentence()` or just `word()` ?

Reply:
yes, I wanted to handle this point in a further commit